python/cucim/MANIFEST.in

```diff
@@ -22,4 +22,6 @@ include tox.ini .travis.yml .appveyor.yml .readthedocs.yml
 include versioneer.py
 include src/cucim/clara/*.so*
 
+recursive-include src/cucim *.pyx *.pxd *.pxi *.py *.pyi *.h *.ini *.npy *.txt *.in *.md
```

Comment:
This is a good idea to include. Thanks for adding 🙏

Should we add the `.cpp` files too?

Reply:
I don't think there are any .cpp files under the Python subfolder, but we do have same .cu files.

Let me add `*.cu` here

Reply:
could remove `*.pyx *.pxd *.pxi` since we don't have any Cython

Reply:
please see latest update